### PR TITLE
Add fallback to v1 API & Fix some tests

### PR DESCRIPTION
**Pull request by marciogranzotto**

### CODE REVIEW

File: wideq/core.py [hunk 1]
```diff
@@ -388,7 +389,10 @@ def discover(cls, country, language) -> "Gateway":
             RequestMethod.GET,
             GATEWAY_URL,
             {"countryCode": country, "langCode": language},
+            country=country,
+            language=language,
         )
+        print(json.dumps(gw))
```
Comment (marciogranzotto) [inline wideq/core.py]:
```suggestion
```

File: wideq/core.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 import datetime
 import requests
 import logging
+import json
```
Comment (marciogranzotto) [inline wideq/core.py]:
```suggestion
```

**Review by no2chem — APPROVED** [review]
Thanks! This looks good to me, but I haven't tested it locally yet.

**Comment by marciogranzotto** [comment]
@no2chem do you think we can merge this?

**Comment by no2chem** [comment]
@marciogranzotto sorry, yes, merged.

**Pull request merged**